package.json

```diff
@@ -9,15 +9,24 @@
     "url": "https://github.com/potaorg/pota"
   },
   "type": "module",
-  "types": "./ts.pota.d.ts",
   "main": "./src/exports.js",
-  "module": "./src/exports.js",
+  "// Using 'typesVersions' here is the only way we could figure out how to get types working for imports of any subpath without any of the problems other approaches have when not using modeResolution:NodeNext (listed in https://stackoverflow.com/questions/77856692/how-to-publish-plain-jsjsdoc-library-for-typescript-consumers)": "",
+  "typesVersions": {
+    "*": {
+      "src/*": ["types/*"]
+    }
+  },
+  "// These exports require moduleResolution:NodeNext to be enabled in the consumer.": "",
   "exports": {
-    "./*": null,
+    "./*": "./*",
     ".": {
-      "types": "./types/src/exports.d.ts",
+      "types": "./types/exports.d.ts",
       "default": "./src/exports.js"
     },
+    "./src/*": {
+      "types": "./types/*",
+      "default": "./src/*"
+    },
```

Comment:
When `moduleResolution:NodeNext` is enabled, the above `typesVersions` is ignored, and instead this rule takes over.